dkg/disk.go

```diff
@@ -34,8 +35,14 @@ func loadDefinition(ctx context.Context, conf Config) (cluster.Definition, error
 
 	// Fetch definition from URI or disk
 
+	parsedURL, err := url.ParseRequestURI(conf.DefFile)
+
 	var def cluster.Definition
-	if validURI(conf.DefFile) {
+	if err == nil && parsedURL.Host != "" {
+		if strings.HasPrefix(parsedURL.Scheme, "https") {
```

Comment:
Reversed the condition?

Reply:
So, we have in the validURI function:
```go
...
return err == nil && (u.Scheme == "http" || u.Scheme == "https") && u.Host != ""
```

The changes in this PR just uses this condition except the `u.Scheme` condition as that is required for adding a warn log logic

Reply:
I believe @pinebit is referring to the HasPrefix check

Reply:
okay, great

apologies, i misunderstood